backend/capellacollab/users/tokens/models.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import datetime
+
+import sqlalchemy as sa
+from sqlalchemy import orm
+
+from capellacollab.core import database
+
+
+class DatabaseUserTokenModel(database.Base):
+    __tablename__ = "basic_auth_token"
+
+    id: orm.Mapped[int] = orm.mapped_column(
+        primary_key=True, index=True, autoincrement=True
+    )
+    user_id: orm.Mapped[int] = orm.mapped_column(sa.ForeignKey("users.id"))
```

Comment:
I put this in the description of the table `sa.ForeignKeyConstraint(["user_id"], ["users.id"], ondelete="CASCADE"),` but it does not work - even though reading the [documentation](https://docs.sqlalchemy.org/en/20/core/constraints.html#sqlalchemy.schema.ForeignKeyConstraint.__init__) I thought it could - do you have an idea how I fix this?

Reply:
done thanks